src/model/Space.hpp

```diff
@@ -114,12 +114,24 @@ namespace model {
 
     void resetPartofTotalFloorArea();
 
+    bool setCeilingHeight(double ceilingHeight);
+
+    void autocalculateCeilingHeight();
+
+    void resetCeilingHeight();
```

Comment:
`isCeilingHeightAutocalculated()` is defined below, ok.

Do we need `boost::optional<double> autocalculatedCeilingHeight() const;` (SQL query) ?

Reply:
Using `getAutosizedValue`? Are autocalculated values populated in the Component Sizing Information table? (Appears there is no precedent for this.)

Reply:
Looks like 8611 has not been addressed. Do you know where in the SQL to find these autocalculated values?

Reply:
@joseph-robertson looking at the E+ source code, as well as testing the 5ZoneAirCooledWithSpaces.idf, the space volume and ceiling height are NOT output anywhere.

The Ceiling Height and Volume for Zones are in the `Initialization Summary` > `Zone Information` report but an equivalent report doesn't exist for Space (yet). (The Volume for a Zone is also in like 2 or 3 other places including the Zone Summary report)

Reply:
Marking as resolved as a result.